src/scss/landing.scss

```diff
@@ -0,0 +1,58 @@
+@import "variables";
+@import "shadow";
+
+.landing-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.landing-body {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 100%;
+  max-width: 1500px;
+  max-height: 800px;
+  padding: 40px;
```

Comment:
Might be worth setting either `margin: 40px` or `box-sizing: border-box` to prevent unnecessary y overflow.